### PR TITLE
Add view for internal transactions error

### LINKED ISSUE #4974: Do not set `error` status for the whole transaction if internal transaction failed
If an internal transaction has an error, do not apply this status to the whole transaction. Instead, display error status near corresponding internal transaction.

A current example of incorrect UX: https://blockscout.com/xdai/mainnet/tx/0x234f036d863a4da9a0a76cc128edfb492be2b77f7d6484e58bf5fb418f8a7b68

The whole transaction marked as reverted:

Screenshot 2021-12-03 at 21 08 26

Though that transaction was successful and only one internal transaction reverted according to: https://dashboard.tenderly.co/tx/xdai/0x234f036d863a4da9a0a76cc128edfb492be2b77f7d6484e58bf5fb418f8a7b68

Thus, if one of the internal transactions has an error and the whole transaction is successful:
- We should remove an error status for the whole transaction transaction
- Transaction tile should not be in red color. Add exclamation point icon with the hint "Error in Internal transaction":

Screenshot 2021-12-03 at 21 31 08

- Whereas errored internal transaction tile should be in red color:
Screenshot 2021-12-03 at 21 16 38

### PR DESCRIPTION
**Pull request by nikitosing**
Close #4974 

### Enhancements

image
image

## Checklist for your Pull Request (PR)

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [x] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [ ] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [ ] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

File: apps/explorer/priv/repo/migrations/20211217201759_add_has_error_in_iternal_txs_field_to_transaction.exs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+defmodule Explorer.Repo.Migrations.AddHasErrorInIternalTxsFieldToTransaction do
+  use Ecto.Migration
+
+  def change do
+    alter table(:transactions) do
+      add(:has_error_in_iternal_txs, :boolean, null: true)
```
Comment (vbaranov) [inline apps/explorer/priv/repo/migrations/20211217201759_add_has_error_in_iternal_txs_field_to_transaction.exs:6]: iternal -> internal

**Review by vbaranov — CHANGES_REQUESTED** [review]

File: apps/explorer/priv/repo/migrations/20211217201759_add_has_error_in_iternal_txs_field_to_transaction.exs (hunk 1 above)
Comment (nikitosing) [reply apps/explorer/priv/repo/migrations/20211217201759_add_has_error_in_iternal_txs_field_to_transaction.exs]: thanks, fixed

File: apps/explorer/priv/repo/migrations/20211217201759_add_has_error_in_iternal_txs_field_to_transaction.exs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+defmodule Explorer.Repo.Migrations.AddHasErrorInIternalTxsFieldToTransaction do
```
Comment (vbaranov) [inline apps/explorer/priv/repo/migrations/20211217201759_add_has_error_in_iternal_txs_field_to_transaction.exs:1]: Iternal -> Internal and also typo in the file name

**Review by vbaranov — CHANGES_REQUESTED** [review]

File: apps/explorer/priv/repo/migrations/20211217201759_add_has_error_in_iternal_txs_field_to_transaction.exs (hunk 2 above)
Comment (nikitosing) [reply apps/explorer/priv/repo/migrations/20211217201759_add_has_error_in_iternal_txs_field_to_transaction.exs]: Thanks

**Review by vbaranov — APPROVED** [review]

**Pull request merged**